cmd/kubeadm/app/cmd/util/documentation.go

```diff
@@ -25,11 +25,6 @@ var (
 	AlphaDisclaimer = `
 		Alpha Disclaimer: this command is currently alpha.
 	`
-
-	// MacroCommandLongDescription provide a standard description for "macro" commands
-	MacroCommandLongDescription = LongDesc(`
-		This command is not meant to be run on its own. See list of available subcommands.
```

Comment:
@SataQiu re: 'upgrade apply' phases, your PR can hopefully merge first and will make this PR to remove also the MacroCommandLongDescription from new phases.

/hold

Reply:
> @SataQiu re: 'upgrade apply' phases, your PR can hopefully merge first and will make this PR to remove also the MacroCommandLongDescription from new phases.
> 
> /hold

seems no longer needed. the macro description is removed without conflicts.